Sage/carbon
# DLS Article

This PR adds a new alert banner to the homepage of the Carbon site, which itself takes you to the first News article talking about the upcoming improvements to Carbon with the Sage Design System.

# Screenshots

### Homepage

![screenshot 2018-11-05 at 13 43 50](https://user-images.githubusercontent.com/2411024/48001588-e8654280-e100-11e8-9720-32a9f451e3ad.png)

### Sage Design System - News Article

![screenshot 2018-11-05 at 13 44 04](https://user-images.githubusercontent.com/2411024/48001590-ebf8c980-e100-11e8-9b10-6c7b35f84c6f.png)

# Testing Instructions

You will need to run the site and view the homepage and the article page. They both should render and work responsively.
